utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/hostapd_stations.lua

```diff
@@ -1,14 +1,42 @@
 local ubus = require "ubus"
 local bit32 = require "bit32"
 
-local function get_wifi_interfaces()
+local function freq_to_chan(freq)
+  if freq == 2484 then
+     return 14
+   elseif freq < 2484 then
+     return (freq - 2407) / 5
+   elseif (freq >= 4910 and freq <= 4980) then
+     return (freq - 4000) / 5
+   elseif (freq <= 45000) then
+     return (freq - 5000) / 5
+   elseif (freq >= 58320 and freq <= 64800) then
+     return (freq - 56160) / 2160
+   else
+     return 0
+   end
+end
+
+local function get_wifi_interface_labels()
   local u = ubus.connect()
   local status = u:call("network.wireless", "status", {})
   local interfaces = {}
 
   for _, dev_table in pairs(status) do
     for _, intf in ipairs(dev_table['interfaces']) do
-      table.insert(interfaces, intf['ifname'])
+      local cfg = intf['config']
+      local hostapd_clients = u:call("hostapd." .. cfg['ifname'], "get_clients", {})
+      local labels = {
+        ifname = cfg['ifname'],
+        ssid = cfg['ssid'],
+        bssid = cfg['macaddr'],
```

Comment:
That is only available, when it is actually set in the config. So I feel we should probably look at `hostapd_cli -i $ifname status` and parse more information from there.